clarifai/client/app.py

```diff
@@ -352,33 +352,12 @@ def create_model(self, model_id: str, **kwargs) -> Model:
 
     return Model.from_auth_helper(auth=self.auth_helper, **kwargs)
 
-  def create_workflow(self,
-                      config_filepath: str,
-                      generate_new_id: bool = False,
-                      display: bool = True) -> Workflow:
-    """Creates a workflow for the app.
-
-    Args:
-        config_filepath (str): The path to the yaml workflow config file.
-        generate_new_id (bool): If True, generate a new workflow ID.
-        display (bool): If True, display the workflow nodes tree.
-
-    Returns:
-        Workflow: A Workflow object for the specified workflow config.
-
-    Example:
-        >>> from clarifai.client.app import App
-        >>> app = App(app_id="app_id", user_id="user_id")
-        >>> workflow = app.create_workflow(config_filepath="config.yml")
-    """
-    if not os.path.exists(config_filepath):
-      raise UserError(f"Workflow config file not found at {config_filepath}")
-
+  def process_workflow_config(self, config_filepath: str):
```

Comment:
```suggestion
  def _process_workflow_config(self, config_filepath: str):
```
can we make this internal function?